.travis.yml

```diff
@@ -14,52 +14,56 @@
 ############################################################################
 
 language: php
-
-php:
-  - 7.1
-
 sudo: false
-
 cache:
-  directories:
-    - .phpunit
-    - $HOME/.composer/cache/files
-
+    directories:
+        - $HOME/.composer/cache/files
+        - $HOME/symfony-bridge/.phpunit
 env:
-  matrix: SYMFONY_VERSION=4.0.*
-  global:
-    - SYMFONY_DEPRECATIONS_HELPER="/.*each.*/"
-    - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
-    - KERNEL_CLASS=Symfony\Cmf\Bundle\CoreBundle\Tests\Fixtures\App\Kernel
-    - SYMFONY_PHPUNIT_VERSION=5.7
-    - TEST_INSTALLATION=false
+    global:
+        - PHPUNIT_FLAGS="-v"
+        - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
+        - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
+        - SYMFONY_DEPRECATIONS_HELPER=/.*each.*/
+        - KERNEL_CLASS=Symfony\Cmf\Bundle\CoreBundle\Tests\Fixtures\App\Kernel
+        - SYMFONY_PHPUNIT_VERSION=5.7
+        - TEST_INSTALLATION=false
 
 matrix:
-  include:
-    - php: 7.2
-      env: STABILITY=dev SYMFONY_VERSION=4.0.*
-    - php: 7.1
-      env: STABILITY=dev COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 7.2
-      env: STABILITY=dev SYMFONY_VERSION=3.3.*
-    - php: 7.2
-      env: STABILITY=dev SYMFONY_VERSION=3.4.*
-    - env: TEST_INSTALLATION=true
-  fast_finish: true
-  allow_failures:
-    - env: TEST_INSTALLATION=true
+    fast_finish: true
+    include:
+        - php: 7.2
+          env:  COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER=weak_vendors
```

Comment:
i am confused why SYMFONY_VERSION=2.8.* is now again not needed. 

but for future proof, can you please add that this gets a `DEPENDENCIES="symfony/symfony:2.8.0` (where 2.8.0 is the lowest supported version of the repo?)

Reply:
I used https://symfony.com/doc/current/bundles/best_practices.html#continuous-integration as a blue print. Maybe not for the lowest, but for the current versions it defenetly makes sense as we won't install `symfony/symfony`anymore.